### PR TITLE
chore: fix build [skip-validation]

### PR DESCRIPTION
**Pull request by peternhale**
Build was failing due to compile error on a mismatched type in tests
[skip-validate-pr]

### CODE REVIEW

File: packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts [hunk 1]
```diff
@@ -87,49 +90,55 @@ class MockEnvironmentVariableCollection
...
-        name: 'salesforcedx-vscode-lwc',
-        serverPath: [
-          'node_modules',
-          '@salesforce',
-          'lwc-language-server',
-          'lib',
-          'server.js'
-        ]
+        aiKey: 'aabbccdd',
+        name: 'salesforcedx-vscode-core',
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts:109]: why core and not lwc if this is inside lwc package?

File: packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts [hunk 2]
```diff
@@ -87,49 +90,55 @@ class MockEnvironmentVariableCollection
...
       }
     } as any;
   }
-  secrets: SecretStorage;
-  extension: Extension<any>;
-  public storageUri: Uri | undefined;
-  public globalStorageUri: Uri = Uri.parse('file://globalStorage');
-  public logUri = Uri.parse('file://logUri');
+  public asAbsolutePath(relativePath: string): string {
+    return path.join('../../../package.json'); // this should point to the src/package.json
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts:115]: and why the change from `../../package.json` to `../../../package.json`?

File: packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts [hunk 3]
```diff
@@ -87,49 +90,55 @@ class MockEnvironmentVariableCollection
...
+      // Implement your logic here
+      // For example, return true, false, or undefined based on some condition
+      return true; // or false or undefined
+    }
+  };
   public logPath = 'logPath';
-  public asAbsolutePath(relativePath: string): string {
-    return path.join('../../package.json'); // this should point to the src/package.json
-  }
+  public logUri = Uri.parse('file://logs');
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts:137]: and why is this `file://logs` and not `file://logUri` as before?

File: packages/salesforcedx-vscode-core/test/vscode-integration/telemetry/index.test.ts [hunk 4]
```diff
@@ -141,7 +141,7 @@ describe('Telemetry', () => {
       const telemetryEnabled = await telemetryService.isTelemetryEnabled();
       expect(telemetryEnabled).to.be.eql(true);
 
-      showTelemetryMessage(mockExtensionContext);
+      await showTelemetryMessage(mockExtensionContext);
```
Comment (daphne-sfdc) [inline packages/salesforcedx-vscode-core/test/vscode-integration/telemetry/index.test.ts:144]: Can you make the same change to all 4 occurrences of `showTelemetryMessage()` in this file?

**Review by CristiCanizales — APPROVED** [review]
LGTM!

**Review by daphne-sfdc — APPROVED** [review]
Approved! 🎉 

Thanks Pete for fixing this compilation error!
